.github/workflows/images-legacy-releases.yaml

```diff
@@ -16,61 +16,80 @@ jobs:
         include:
           - name: cilium
             dockerfile: ./Dockerfile
+
           - name: operator
             dockerfile: ./cilium-operator.Dockerfile
+
           - name: operator-aws
             dockerfile: ./cilium-operator-aws.Dockerfile
+
           - name: operator-azure
             dockerfile: ./cilium-operator-azure.Dockerfile
+
           - name: operator-generic
             dockerfile: ./cilium-operator-generic.Dockerfile
+
           - name: hubble-relay
             dockerfile: ./hubble-relay.Dockerfile
+
           - name: clustermesh-apiserver
             dockerfile: ./clustermesh-apiserver.Dockerfile
+
           - name: docker-plugin
             dockerfile: ./cilium-docker-plugin.Dockerfile
+
     steps:
-      - name: Set output
-        id: vars
-        run: echo ::set-output name=tag::${GITHUB_REF##*/}
       - name: Set up Docker Buildx
         uses: docker/setup-buildx-action@154c24e1f33dbb5865a021c99f1318cfebf27b32 # v1.1.1
 
-      - name: Checkout Stable Branch Source Code
-        uses: actions/checkout@5a4ac9002d0be2fb38bd78e4b4dbde5606d7042f # v2.3.4
       - name: Login to DockerHub
         uses: docker/login-action@f3364599c6aa293cdc2b8391b1b56d0c30e45c8a # v1.8.0
         with:
           username: ${{ secrets.DOCKER_HUB_RELEASE_USERNAME }}
           password: ${{ secrets.DOCKER_HUB_RELEASE_PASSWORD }}
+
       - name: Login to quay.io
         uses: docker/login-action@f3364599c6aa293cdc2b8391b1b56d0c30e45c8a # v1.8.0
         with:
           registry: quay.io
           username: ${{ secrets.QUAY_USERNAME_RELEASE_USERNAME }}
           password: ${{ secrets.QUAY_PASSWORD_RELEASE_PASSWORD }}
+
+      - name: Getting image tag
+        id: tag
+        run: |
+          echo ::set-output name=tag::${GITHUB_REF##*/}
+
+      - name: Checkout Source Code
+        uses: actions/checkout@5a4ac9002d0be2fb38bd78e4b4dbde5606d7042f # v2.3.4
+
       - name: Release Build ${{ matrix.name }}
-        uses: docker/build-push-action@4a531fa5a603bab87dfa56578bd82b28508c9547 # v2.2.2
+        uses: docker/build-push-action@9379083e426e2e84abb80c8c091f5cdeb7d3fd7a # v2.2.2
```

Comment:
Is this still `v2.2.2`? The sha changed.

Reply:
Not it is not, I need to remove all comments sense dependabot will update those SHAs automatically.